pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -118,13 +125,13 @@ buildPythonApplication rec {
       make docs
       ${python.interpreter} setup.py kitty.app ${darwinOptions}
     '' else ''
-      ${python.interpreter} setup.py build-launcher
       ${python.interpreter} setup.py linux-package \
       --egl-library='${lib.getLib libGL}/lib/libEGL.so.1' \
       --startup-notification-library='${libstartup_notification}/lib/libstartup-notification-1.so' \
       --canberra-library='${libcanberra}/lib/libcanberra.so' \
       --fontconfig-library='${fontconfig.lib}/lib/libfontconfig.so' \
       ${commonOptions}
+      ${python.interpreter} setup.py build-launcher
```

Comment:
The launcher needs to be built after the package and is required for the tests to run.

Reply:
```suggestion
      ${python.ForBuild.interpreter} setup.py build-launcher
```